services/frontend/src/views/NewAnalysis.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div id="new-analysis">
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'NewAnalysis',
+  components: {
+  },
+}
+</script>
+
+<style>
+#new-analysis {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
Refactor this duplicated style into a shared CSS class with a proper name

Reply:
Did that and some other frequent duplications.